frontend/packages/ceph-storage-plugin/src/components/converged-credentials/credentials.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { k8sGet } from '@console/internal/module/k8s';
+import { SecretModel, ConfigMapModel } from '@console/internal/models';
+import { createModalLauncher, ModalComponentProps } from '@console/internal/components/factory';
+import { OCS_NS } from '../../constants';
+
+const parseMonEndpoints = (data: string): string => {
```

Comment:
I dont think it deserves to be a separate function as you call it only in one place. Why not just inline it ?

Reply:
ack.